app-server/app/src/main/scala/user/User.scala

```diff
@@ -2,40 +2,63 @@ package com.azavea.rf.user
 
 import scala.concurrent.{Future, ExecutionContext}
 import scala.util.{Try, Success, Failure}
+import scala.language.postfixOps
 import java.sql.Timestamp
-import slick.lifted.{ColumnOrdered, TableQuery, Query}
+import slick.lifted.Query
 import com.lonelyplanet.akka.http.extensions.{PageRequest, Order}
-import de.choffmeister.auth.common.JsonWebToken
 
-import com.azavea.rf.datamodel.latest.schema.tables.{Users, Organizations}
-import com.azavea.rf.datamodel.latest.schema.tables.UsersRow
+import com.azavea.rf.datamodel.latest.schema.tables.{Users, Organizations, UsersToOrganizations}
+import com.azavea.rf.datamodel.latest.schema.tables.{UsersRow, UsersToOrganizationsRow}
 import com.azavea.rf.utils.{Database,PaginatedResponse}
 
 
+class UserErrorException(message: String = null, cause: Throwable = null) extends RuntimeException(UserErrorException.defaultMessage(message, cause), cause)
+object UserErrorException {
```

Comment:
Spacing here is off and confusing -- you can put `RuntimeException` on the second line and there needs to be 2 lines between the class and object here